traefik.go

```diff
@@ -266,8 +268,12 @@ func run(traefikConfiguration *TraefikConfiguration) {
 	}
 	log.Debugf("Global configuration loaded %s", string(jsonConf))
 	server := NewServer(globalConfiguration)
-	server.Start()
 	defer server.Close()
+	sent, err := daemon.SdNotify("READY=1")
```

Comment:
hopefully tests was there to protect us, but @emilevauge eyes are quite faster ;)